### PR TITLE
Alternative playback engines can  be defined based on collectionid.

### PR DESCRIPTION
**Pull request by thomasegense**
Alternative playback engines can  be defined based on collectionid for the document. This requires the WARC files has been indexed with a collectionid.

@jorntx  Wait with merge until Volodymyr Kushnarenko has tested and approved.

### CODE REVIEW

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemStandardInfo.vue [hunk 1]
```diff
@@ -82,10 +82,26 @@ export default {
       return `${configs.playbackConfig.solrwaybackBaseURL}services/viewForward?source_file_path=${source_file_path}&offset=${source_file_offset}`    
     },
 
-    getOpenWaybackLink(wayback_date, url) {
-      return `${configs.playbackConfig.openwaybackBaseURL}${wayback_date}/${url}`    
+    getAlternativePlaybackEngineLink(wayback_date, url, collection) {    
+        //Test if collection specific playback is enabled. 
+        var collectionPlayback=configs.collection.playback.get('PLAYBACK_'+collection)
```
Comment (jorntx) [inline src/js/src/components/searchSingleItemComponents/SearchSingleItemStandardInfo.vue:87]: Change 'var' to 'const'

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemStandardInfo.vue [hunk 2]
```diff
@@ -82,10 +82,26 @@ export default {
...
+        if (collectionPlayback != null){ //Use default playback engine
+         //console.log('collection playback')
+         var collectionPlaybackLink = collectionPlayback+wayback_date+'/'+url
```
Comment (jorntx) [inline src/js/src/components/searchSingleItemComponents/SearchSingleItemStandardInfo.vue:90]: Change 'var' to 'const'

**Review by jorntx — APPROVED** [review]
Small smal change requested but it is so small that I still approve.

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemStandardInfo.vue (hunk 2 above)
Comment (thomasegense) [reply src/js/src/components/searchSingleItemComponents/SearchSingleItemStandardInfo.vue]: fixed

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemStandardInfo.vue (hunk 1 above)
Comment (thomasegense) [reply src/js/src/components/searchSingleItemComponents/SearchSingleItemStandardInfo.vue]: fixed

**Comment by thomasegense** [comment]
@jorntx  fixed

**Comment by jorntx** [comment]
Good - when you give me the go I will merge this branch

**Pull request merged**